assets/javascripts/discourse/templates/admin-plugins/discourse-ai/ai-personas/index.hbs

```diff
@@ -0,0 +1 @@
+<AiPersonaListEditor @personas={{this.model}} />
```

Comment:
I am not sure how to structure it...but I think there is a way you can make these routes using `{{outlet}}` that means you won't have to make multiple templates and repeat this component. Might be worth asking someone like CvX about it, but it works as it is too.

Reply:
oh but they all come with a slightly diff flavor, I kind of like this explicitness